src/services/expenses.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { Expense } = require('../models/Expense');
+
+async function getAll() {
+  const result = await Expense.findAll({
+    order: ['spentAt'],
+  });
+
+  return result;
+};
```

Comment:
send a request to receive the necessary data right here (by category, userId, from, to)